R/relabel.R

```diff
@@ -22,10 +22,14 @@
 #'
 #' rincome2 <- fct_relabel(gss_cat$rincome, convert_income)
 #' fct_count(rincome2)
+#'
+#'
```

Comment:
Can you please think through the use of blank lines here? Generally they should be used to group together related code. (This may require tweaking to the entire example block)

Reply:
I changed the new one to pipes and put the function definition at the start of the second one, so now each example has no blank lines within it, and there's one between them. The `fct_count(gss_cat$rincome)` doesn't _have_ to come after the function definition, but I think it's more readable in terms of whitespace.